docs/root/start/examples/hello_world.rst

```diff
@@ -32,7 +32,7 @@ The demo is available below (along with building instructions) in the following
 Java
 ----
 
-First, build the :ref:`android_aar` artifact (or download it from one of the recent releases_).
+First, build the :ref:`android_aar` artifact.
```

Comment:
Why remove these? Seems like they're probably still relevant?

Reply:
so that people _have_ to see the instructions about what to do with the download. Can revert if you think that is not necessary.